### DISCUSSION TITLE
Windows RDP - Auto closing session

### DISCUSSION THREAD

**Discussion by notour** [mRemoteNG/mRemoteNG#2485] [Q&A]
Hi,
My team is using mRemoteNG. We have often the issue that to many sessions are active on servers.
Is there a way to auto "Closing the session" (Sign out) when inactivity is detected ?

Thanks

**Comment by LeonS13** [comment]
If you are talking about rdp sessions to Windows machines (both server and client os can do this) then your request should be configured on the machines you connect to remote and not in mRemoteNG. In a professional setting you should do this with group policies. When you do not have a windows domain then you can also do the same with local policies or with registry settings.

**Comment by Kvarkas** [comment]
This could be added by Windows policy, in mRemote you could only add a ps script perhaps what do logoff on disconect, potentialy we could add something like force close but need to play with such to see if that posible
